annotations-impl/src/main/java/org/ocpsoft/rewrite/annotation/scan/WebClassesFinder.java

```diff
@@ -174,13 +174,13 @@ private void handleClassEntry(URL entryUrl, String entryName, ClassVisitor visit
              */
             try
             {
-               classFileStream = entryUrl.openStream();
+              classFileStream = servletContext.getResourceAsStream(entryName);
             }
-            catch (IOException e)
+            catch (Exception e)
```

Comment:
The try block does not throw the checked exception IOException anymore. To avoid a compile error, it has to be changed.